File: app/src/main/java/com/android/principle/MainActivity.java
package com.android.principle;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    private Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button = findViewById(R.id.okhttp_button);
        button.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.okhttp_button:
//                startActivity(new Intent(this, OkHttpActivity.class));
                break;
        }
    }

}

File: rxandroid/src/main/java/cn/kpa/rxandroid/MainActivity.kt
package cn.kpa.rxandroid

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

/**
 * Activity 充当的角色
 * 1、UI主线程 负责绘制
 * 2、开启子线程 获取数据
 * 3、赋值到控件中
 * 4、逻辑判断等等
 */
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        download.setOnClickListener({

        })
    }
}
